### ISSUE TITLE
Adaptive Loss for Boosting Algorithms

### ISSUE THREAD

**Issue by fsaforo1** [jonbarron/robust_loss_pytorch#35]
Jon, can you provide a tutorial (or code snippet) on how to use this tool for boosting algorithms like LightGBM

**Comment by jonbarron** [comment]
Sorry, I have no idea how this would be done. Boosting algorithms are usually not agnostic to the choice of loss function.
